.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - main
+
+defaults:
+  run:
+    shell: bash --noprofile --norc -euo pipefail {0}
+
+env:
+  # Use Maven wrapper from repo with Maven version and other configs
+  MAVEN: ./mvnw -B
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v3
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
```

Comment:
I thought Trino was using Zulu - is that not the case?

Reply:
But let me switch to Zulu.

Reply:
yeah i agree it shouldn't matter - just trying to keep things uniform